docker/entrypoint.sh

```diff
@@ -272,6 +275,18 @@ case $1 in
     PACKAGE_PATH=${WORKDIR}/${PACKAGE_NAME}/src/.libs/${PACKAGE_NAME}.so
     CONF_PATH=${WORKDIR}/${PACKAGE_NAME}/query.conf
 
+    mkdir -p /usr/lib/zabbix/modules 
```

Comment:
this looks fine, but the installation stuff probably belongs in the Dockerfile rather than being called at runtime. I'm reworking some of this in my libzbxpython project so will patch this up later.